src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -376,4 +369,27 @@ class EthService(
     }
   }
 
+  private def doCall(req: CallRequest): Either[JsonRpcError, Ledger.TxResult] = {
+    val fromAddress = req.tx.from
+      .map(Address.apply) // `from` param, if specified
+      .getOrElse(
+      keyStore
+        .listAccounts().getOrElse(Nil).headOption // first account, if exists and `from` param not specified
+        .getOrElse(Address(0))) // 0x0 default
+
+    val toAddress = req.tx.to.map(Address.apply)
+
+    val gasLimit = req.tx.gas.getOrElse {
+      resolveBlock(BlockParam.Latest) match {
+        case Right(b) => b.header.gasLimit
+        case Left(err) => BigInt(0)
+      }
+    }
+    val tx = Transaction(0, req.tx.gasPrice, gasLimit, toAddress, req.tx.value, req.tx.data)
+    val stx = SignedTransaction(tx, ECDSASignature(0, 0, 0.toByte), fromAddress)
```

Comment:
`ECDSASignature(0, 0, 0.toByte)` <- this is not obviouse and at first glance looks strange, maybe we can extract this to val and add name
something like `fakeSignature` ?